base_classes/nyaml/NXpositioner.yaml

```diff
@@ -53,22 +64,10 @@ NXpositioner(NXobject):
     unit: NX_ANY
     doc: |
       time to ramp the velocity up to full speed
-  
   # TODO other parameters: settling time, backlash, link to readback channel
   controller_record:
     doc: |
-      Hardware device record, e.g. EPICS process variable, taco/tango ...
-  \@default:
```

Comment:
it can be kept

Reply:
The default attribute is there. Move it right after the base class definition.